ios/Exponent/Versioned/Core/UniversalModules/EXFacebook/EXScopedFacebook.m

```diff
@@ -48,21 +48,16 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId andParams:(NSDicti
     BOOL hasPreviouslySetAutoInitEnabled = [_settings boolForKey:AUTO_INIT_KEY];
     BOOL manifestDefinesAutoInitEnabled = [params[@"manifest"][@"facebookAutoInitEnabled"] boolValue];
 
-    NSString *facebookAppId = params[@"manifest"][@"facebookAppId"];
+    NSString *scopedFacebookAppId = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"FacebookAppID"];
     NSString *facebookDisplayName = params[@"manifest"][@"facebookDisplayName"];
 
     if (hasPreviouslySetAutoInitEnabled || manifestDefinesAutoInitEnabled) {
-      // FacebookAppId is a prerequisite for initialization.
       // This happens even before the app foregrounds, which mimics
       // the mechanism behind EXFacebookAppDelegate.
-      if (facebookAppId) {
```

Comment:
Could someone build an ad-hoc Expo Client without providing a `FacebookAppID`? If so, is it safe to remove the `if (facebookAppId)`?

Reply:
added this back, although i need to double-check the ad-hoc client generation code to see if we assign the given facebookAppId (from app.json) to the info.plist

Reply:
Don't forget to add this back to the versioned `EXScopedFacebook` too! 😉